internal/util/app.go

```diff
@@ -62,9 +69,21 @@ func AppShortVersion() string {
 }
 
 var getPdpID = sync.OnceValue(func() string {
-	//nolint:gosec
-	nodeID := md5.Sum(uuid.NodeID())
-	return fmt.Sprintf("%X-%d", nodeID, os.Getpid())
+	machineID, err := machineid.ID()
```

Comment:
Wouldn't it be easier to use [machineid.ProtectedID(appID string)](https://github.com/keygen-sh/machineid/blob/0bd4ea32a95c643d8bdf26ed4a2f29bf70c322a0/id.go#L39-L45)?